topics/transcriptomics/tutorials/scrna-plant/tutorial.md

```diff
@@ -0,0 +1,533 @@
+>        - *"The size of local neighborhood (in terms of number of neighboring data points) used for manifold approximation"*: `10`
+>        - *"Number of PCs to use"*: `40`
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > UMAP relies on a connected graph of cells to operate. Please see: [An introduction to scRNA-seq data analysis]({% link videos/#video-transcriptomics-scrna-intro %}) at time point 13:40 for more information on how this process works.
+>    {: .comment} 
+>
+> 1. {% tool [Cluster, infer trajectories and embed](toolshed.g2.bx.psu.edu/repos/iuc/scanpy_cluster_reduce_dimension/scanpy_cluster_reduce_dimension/1.7.1+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `anndata_out` (output of **Inspect and manipulate** {% icon tool %})
+>    - *"Method used"*: `Embed the neighborhood graph using UMAP, using 'tl.umap'`
+>
+{: .hands_on}
+
+With our data now sufficiently "flat" and ready for human consumption, we can now do a naive plot of both the PCA and the UMAP and show cell assignments coloured by batch.
+
+> ### {% icon hands_on %} Hands-on: Plot the PCA and UMAP by Batch
+>
+> 1. {% tool [Plot with scanpy](toolshed.g2.bx.psu.edu/repos/iuc/scanpy_plot/scanpy_plot/1.7.1+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `anndata_out` (output of **Cluster, infer trajectories and embed** {% icon tool %})
+>    - *"Method used for plotting"*: `Embeddings: Scatter plot in PCA coordinates, using 'pl.pca`
+>      - *"Keys for annotations of observations/cells or variables/genes"*: `batch`
+>      - In *"Plot attributes"*
+>        - *"Colors to use for plotting categorical annotation groups"*: `rainbow`
+>
+> 1. {% tool [Plot with scanpy](toolshed.g2.bx.psu.edu/repos/iuc/scanpy_plot/scanpy_plot/1.7.1+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Annotated data matrix"*: `anndata_out` (output of **Cluster, infer trajectories and embed** {% icon tool %})
+>    - *"Method used for plotting"*: `Embeddings: Scatter plot in UMAP basis, using 'pl.umap'`
+>      - *"Keys for annotations of observations/cells or variables/genes"*: `batch`
+>      - In *"Plot attributes"*
+>        - *"Colors to use for plotting categorical annotation groups"*: `rainbow`
+>
+{: .hands_on}
+
+| PCA                                | UMAP                                 |
+|:----------------------------------:|:------------------------------------:|
+| ![PCA](../../images/pca_batch.png) | ![UMAP](../../images/umap_batch.png) |
+
+
+From this we can see that there is a reasonable overlap in our batches shown both in the PCA and UMAP embeddings. This is good because it shows that though there is *some* batch effect (i.e. cells from one batch appear to cluster on a different side of the plot than the other) it is not significant enough for there not to be some commonality between the batches. 
```

Comment:
```suggestion
From this, we can see that there is a reasonable overlap in our batches shown both in the PCA and UMAP embeddings. This is good because it shows that though there is *some* batch effect (i.e. cells from one batch appear to cluster on a different side of the plot than the other) it is not significant enough for there not to be some commonality between the batches. 
```